sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClientJavaDocCodeSamples.java

```diff
@@ -0,0 +1,122 @@
+     */
+    public void instantiate() {
+        // BEGIN: com.azure.messaging.servicebus.servicebusasyncsenderclient.instantiation
+        // The required parameter is a way to authenticate with Service Bus using credentials.
+        // The connectionString provides a way to authenticate with Service Bus.
+        ServiceBusSenderAsyncClient sender = new ServiceBusClientBuilder()
+            .connectionString(
+                "Endpoint={fully-qualified-namespace};SharedAccessKeyName={policy-name};SharedAccessKey={key}")
+            .buildSenderClientBuilder()
+            .entityName("<QUEUE-NAME>")
+            .buildAsyncClient();
+        // END: com.azure.messaging.servicebus.servicebusasyncsenderclient.instantiation
+
+        sender.close();
+    }
+
+    /**
+     * Code snippet demonstrating how to send a batch to Service Bus queue or topic.
+     */
+    public void sendBatch() {
+        // BEGIN: com.azure.messaging.servicebus.servicebusasyncsenderclient.createBatch
+        // The required parameter is a way to authenticate with Service Bus using credentials.
+        // The connectionString provides a way to authenticate with Service Bus.
+        ServiceBusSenderAsyncClient sender = new ServiceBusClientBuilder()
+            .connectionString(
+                "Endpoint={fully-qualified-namespace};SharedAccessKeyName={policy-name};SharedAccessKey={key}")
+            .buildSenderClientBuilder()
+            .entityName("<QUEUE-NAME>")
+            .buildAsyncClient();
+
+        // Creating a batch without options set, will allow for automatic routing of events to any partition.
+        sender.createBatch().flatMap(batch -> {
+            batch.tryAdd(new ServiceBusMessage("test-1".getBytes(UTF_8)));
+            batch.tryAdd(new ServiceBusMessage("test-2".getBytes(UTF_8)));
+            return sender.send(batch);
+        }).subscribe(unused -> {
+
+        },
+            error -> System.err.println("Error occurred while sending batch:" + error),
+            () -> System.out.println("Send complete."));
+        // END: com.azure.messaging.servicebus.servicebusasyncsenderclient.createBatch
+
+        sender.close();
+    }
+
+
+    /**
+     * Code snippet demonstrating how to create a size-limited {@link ServiceBusMessageBatch} and send it.
+     */
+    public void batchSizeLimited() {
+        final ServiceBusSenderAsyncClient sender = new ServiceBusClientBuilder()
+            .buildSenderClientBuilder()
+            .buildAsyncClient();
+
+        final ServiceBusMessage firstMessage = new ServiceBusMessage("92".getBytes(UTF_8));
+        firstMessage.getProperties().put("telemetry", "latency");
+        final ServiceBusMessage secondMessage = new ServiceBusMessage("98".getBytes(UTF_8));
+        secondMessage.getProperties().put("telemetry", "cpu-temperature");
+
+        // BEGIN: com.azure.messaging.servicebus.servicebusasyncsenderclient.createBatch#CreateBatchOptions-int
```

Comment:
is there a createBatch that takes an int parameter? I think the snippet text is wrong